### PR TITLE
feat: add avatars options

### PR DESCRIPTION
**Pull request by instalando**

### CODE REVIEW

**Review by JaisonPeres — APPROVED** [review]

**Review by afonsovinicius — APPROVED** [review]

File: src/services/video-conference-manager/types.ts [hunk 1]
```diff
@@ -44,6 +45,11 @@ export interface Locale {
   };
 }
 
+export interface Avatar {
+  thumb: string;
+  link: string;
+}
+
```
Comment (carlossantos74) [inline src/services/video-conference-manager/types.ts:52]: You don't need to create this interface here, there is already an `Avatar` interface that you can use

Comment (carlossantos74) [reply src/services/video-conference-manager/types.ts:52]: https://github.com/SuperViz/sdk/blob/6e5e6ae210389de959057dd32b235215d89fe16b/src/common/types/user.types.ts#L16

File: src/common/types/events.types.ts [hunk 2]
```diff
@@ -19,6 +19,7 @@ export enum MeetingEvent {
   MEETING_KICK_USERS = 'meeting.kick-all-users',
   MEETING_STATE_UPDATE = 'meeting.state-update',
   MEETING_CONNECTION_STATUS_CHANGE = 'meeting.connection-status-change',
+  MEETING_AVATARS_UPDATE = 'meeting.avatars-update',
```
Comment (carlossantos74) [inline src/common/types/events.types.ts:22]:
```suggestion
  MEETING_AVATAR_LIST_UPDATE = 'meeting.avatar-list-update',
```

File: src/common/types/sdk-options.types.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import type { Locale, Offset } from '../../services/video-conference-manager/types';
+import type { Locale, Avatar, Offset } from '../../services/video-conference-manager/types';
```
Comment (carlossantos74) [inline src/common/types/sdk-options.types.ts:1]: You should use the default interface for avatars. And remove this import.


```suggestion
import type { Locale, Offset } from '../../services/video-conference-manager/types';
```

File: src/services/video-conference-manager/index.ts [hunk 4]
```diff
@@ -317,6 +330,15 @@ export default class VideoConfereceManager {
...
 
+  /**
+   * @function updateMeetingAvatar
+   * @description update list of avatars
+   * @returns {void}
+   */
+  private updateMeetingAvatars = (): void => {
+    this.messageBridge.publish(MeetingEvent.MEETING_AVATARS_UPDATE, this.meetingAvatars);
+  };
+
```
Comment (carlossantos74) [inline src/services/video-conference-manager/index.ts:341]:
```suggestion
    this.messageBridge.publish(MeetingEvent.MEETING_AVATAR_LIST_UPDATE, this.meetingAvatars);
  };

```

**Review by carlossantos74 — CHANGES_REQUESTED** [review]

**Review by carlossantos74 — APPROVED** [review]

**Comment by carlossantos74** [comment]
Great Job 🚀

**Pull request merged**